### DISCUSSION TITLE
Query Regarding Page Navigation in Vike Software

### DISCUSSION THREAD

**Discussion by stskavishkannanm** [vikejs/vike#1598] [Discussion]
I am currently exploring the capabilities of Vike software and have encountered a query regarding page navigation functionality. In Vue.js applications, we typically use the router.push() method to navigate from one page to another programmatically.

Could you please clarify if there is an equivalent functionality or method available in Vike for handling page redirection? I'm particularly interested in how we can programmatically redirect users to different pages within a Vike application.

**Comment by brillout** [comment]
https://vike.dev/navigate

Respect https://brillout.github.io/rules and don't spam. Last warning.
